src/javascripts/jquery.tipsy.js

```diff
@@ -118,6 +118,21 @@
                 }
             }
         },
+
+        destroy: function(){
+          $.removeData(this.$element.get(0));
```

Comment:
I see - I just wondered if this was some specialty to fix the memory leaks you were talking about earlier - in which case I would have voted for adding a comment.